### ISSUE TITLE
None of the link works. Please delete this repo. 

### ISSUE THREAD

**Issue by girisagar46** [mohnkhan/Free-OReilly-Books#3]
OReilly do not provide direct link to pdf, ebub or mobi.

This repo is mis-leading.

**Comment by javaHelper** [comment]
agree, all links are invalid, delete this repo!

**Comment by mohnkhan** [comment]
They used to work earlier, trying to see what has changed mean while.

**Comment by sarnobat** [comment]
I disagree about deleting it. They can still be a starting point to tracking them down yourself (e.g. https://github.com/ffisk/books/blob/master/microservices-for-java-developers.pdf ).

Oh, and besides, it's disrespectful to the author.

**Comment by jcsandoval98** [comment]
dont delete the repo please

**archive.org has the files if you need them**
[example](https://web.archive.org/web/20201022195912/https://www.oreilly.com/data/free/files/industrial-internet.epub)

**Comment by o-200** [comment]
There is an option to parse all content from o'reily API's and make .epub or .pdf file
https://learning.oreilly.com/api/v2/epubs/urn:orm:book:9781835468609/files
